google/cloud/ndb/global_cache.py

```diff
@@ -27,6 +27,10 @@
 import pymemcache
 import redis as redis_module
 
+# Python 2.7 doesn't have ConnectionError. In Python 3, ConnectionError is subclass of
+# OSError, which Python 2.7 does have.
+ConnectionError = getattr(__builtins__, "ConnectionError", OSError)
```

Comment:
Do redis / memchache raise a bare `OSError` on 2.7?

Reply:
I'm not entirely sure. I'll see if I can flush out the actual behavior.